utils/check_repo.py

```diff
@@ -240,6 +240,8 @@
     "VisualBertForMultipleChoice",
     "TFWav2Vec2ForCTC",
     "TFHubertForCTC",
+    "MaskFormerForInstanceSegmentation",
+    "OneFormerForUniversalSegmentation",
```

Comment:
These can both be removed, since they are supported by `AutoModelForUniversalSegmentation`